migration/models/question-type.model.js

```diff
@@ -24,7 +24,7 @@ module.exports = function Table(sequelize, DataTypes) {
                     const names = [
                         'text', 'choice', 'choices', 'bool', 'integer', 'float',
                         'zip', 'date', 'pounds', 'year', 'month', 'day',
-                        'feet-inches', 'blood-pressure', 'choice-ref',
+                        'feet-inches', 'blood-pressure', 'choice-ref', 'scale',
```

Comment:
You do not change files in the migration/models directory. You need to insert scale in your migration script.